src/biz/tugay/groovyship/cli/CliGameController.groovy

```diff
@@ -0,0 +1,107 @@
+package biz.tugay.groovyship.cli
+
+import biz.tugay.groovyship.modal.Board
+import biz.tugay.groovyship.service.GameService
+
+import java.nio.charset.StandardCharsets
+
+import static java.lang.Integer.parseInt
+
+class CliGameController
+{
+
+  static GameService gameService = new GameService()
+
+  static Scanner scanner = new Scanner(System.in, StandardCharsets.UTF_8.toString())
+
+  static BoardCommandLinePrinter boardCommandLinePrinter = new BoardCommandLinePrinter()
+
+  static Board board = null
+
+  static void main(String[] args) {
```

Comment:
This is the entry point if you want to play the game.

Reply:
looking familiar .... even in different languages :)
unless one uses spring, them slightly different